src/index.js

```diff
@@ -40,6 +41,17 @@ class ExtractTextPlugin {
     return { loader: require.resolve('./loader'), options };
   }
 
+  static cloneModule(module) {
```

Comment:
Can we move this in https://github.com/webpack-contrib/extract-text-webpack-plugin/blob/master/src/lib/helpers.js

Reply:
sure